src/client/api/quoteBundleSelectors.ts

```diff
@@ -98,6 +98,10 @@ export const getMainQuote = (bundle: QuoteBundle) => {
   return bundle.quotes[0]
 }
 
+export const getEmail = (bundle: QuoteBundle) => {
+  return bundle.quotes[0].email
```

Comment:
nit: should this compose other selectors like: `getMainQuote` instead of assuming things?

Reply:
I agree, I guess we solve it better in the new acquisition API:s